flatbuffers.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix=@CMAKE_INSTALL_PREFIX@
```

Comment:
Please move this to `CMake` ?

Reply:
How exactly ?

The .pc template is hand-written on purpose (so we specific additions/changes can easily be done anythime when necessary) - it just needs to fill in some built-time defined data (eg. pathes).

Reply:
I meant the `CMake` dir. Less files in the root the better.